Makefile

```
@@ -1,37 +1,75 @@
+include .bingo/Variables.mk
 SHELL=/bin/bash
 
-GODIRS_NOVENDOR   ?= $(shell go list ./... | grep -v /vendor/)
+PROVIDER_MODULES ?= $(shell ls -d $(PWD)/providers/*)
+MODULES          ?= $(PROVIDER_MODULES) $(PWD)/
+
 GOFILES_NOVENDOR  ?= $(shell find . -path ./vendor -prune -o -name '*.go' -print)
 
-GOPATH            ?= $(shell go env GOPATH)
 GOBIN             ?= $(firstword $(subst :, ,${GOPATH}))/bin
+
 GO111MODULE       ?= on
 export GO111MODULE
 GOPROXY           ?= https://proxy.golang.org
 export GOPROXY
 
-# TODO(bwplotka): Auto install using separate go.mod for tools.
-GOIMPORTS         ?= $(GOBIN)/goimports
+.PHONY: fmt
+fmt:
+	@echo "Running fmt for all modules: ${MODULES}"
+	@$(GOIMPORTS) -local github.com/grpc-ecosystem/go-grpc-middleware/v2 -w $(GOFILES_NOVENDOR)
+	@$(foreach dir,$(MODULES),$(MAKE) fmt_module DIR=$(dir))
 
-all: vet fmt test
+.PHONY: fmt_module
+fmt_module:
+	go fmt $(shell cd $(DIR) && go list ./... | grep -v /vendor/)
```

We don't have a vendor directory anymore, maybe we could simplify this?
```suggestion
	gofmt -w -s ./...
```